### PR TITLE
Fixing RHELOSP-176036 and RHELOSP-176038

### PR DESCRIPTION
**Pull request by lnatapov**
Fixing RHELOSP-176036 and RHELOSP-176038

Depends-On: https://github.com/infrawatch/feature-verification-tests/pull/177

### CODE REVIEW

**Comment by lnatapov** [comment]
Recheck

**Review by elfiesmelfie — APPROVED** [review]

**Comment by elfiesmelfie** [comment]
There is no CI coverage for the change, so I'm willing to merge without waiting for CI to pass, since it won't tell us anything about the actual change

File: roles/test_sensubility/tasks/test_health_status.yml [hunk 1]
```diff
@@ -64,14 +66,18 @@
...
+
 
 - name: RHELOSP-176038
   # Description: Check that health status of container changed to 1
   ansible.builtin.shell:
     cmd: >-
       /usr/bin/curl -k {{ prom_auth_string }} \
         -g https://{{ prom_url }}/api/v1/query? \
-        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="{{ groups['overcloud_nodes'][0] }}"}[10m])' \
+        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="controller-0.redhat.local"}[10m])' \
```
Comment (mgirgisf) [inline roles/test_sensubility/tasks/test_health_status.yml:80]: Are we going to have this host always fixed ?

**Comment by elfiesmelfie** [comment]
recheck

**Review by mgirgisf — APPROVED** [review]

**Pull request merged**